docs/admin/kube-apiserver.md

```diff
@@ -82,6 +82,10 @@ kube-apiserver
       --etcd-servers=[]: List of etcd servers to connect with (http://ip:port), comma separated.
       --etcd-servers-overrides=[]: Per-resource etcd servers overrides, comma separated. The individual override format: group/resource#servers, where servers are http://ip:port, semicolon separated.
       --event-ttl=1h0m0s: Amount of time to retain events. Default 1 hour.
```

Comment:
I think, `--experimental-authorization-keystone-role` and `--experimental-authorization-keystone-single-projectid`, these two args can also be included in the Keystone config file. As these args are all passed to `kube-apiserver` when bootstrapping kubernetes env, all the configurations are fixed unless restarting `kube-apiserver`.